diff --git a/dashboard_demo/models/scoreboard.py b/dashboard_demo/models/scoreboard.py
new file mode 100644
index 0000000..a94772b
--- /dev/null
+++ b/dashboard_demo/models/scoreboard.py
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+
+import marshmallow_dataclass
+import marshmallow.validate as mv
+
+
+MonthType = NewType("MonthType", str, mv.Regexp(r'^\d{4}-\d{2}$'))
+
+@dataclass
+class Properties:
+    month: MonthType
+    office: str
+
+@dataclass
+class Household:
+    # TODO: add other fields
+    
+
+@dataclass
+class Scoreboard:
+    record_id: int
+    properties: Properties
+    
